lib/start/src/bottom_stuff.dart

```diff
@@ -50,9 +58,11 @@ class BottomStuff extends HookWidget {
           final double t = controller.value;
 
           final tColumns = (t - 1) * (forwardOrComplete ? 2 : 1) + 1;
-          final tSeparator = forwardOrComplete
-              ? curve.transform(math.min(t * 2, 1))
-              : 1 - curve.transform(1 - t);
+          final tSeparator = math.max(
+              forwardOrComplete
+                  ? curve.transform(math.min(t * 2, 1))
+                  : 1 - curve.transform(1 - t),
+              0.0);
```

Comment:
In almost every case (including this one), `curve.transform()` takes a value between `0.0` and `1.0` and outputs a different value within that same range.

So we probably don't need the `math.max()` function here. (If removing it causes a problem, please let me know, since that means there's a bug somewhere 🙂)